### PR TITLE
Add desktop runner to phantom js

### LINKED ISSUE #136: Add testing for nav (both mobile and non-mobile)
With all the similar but different mobile nav features we should really be testing the navigation menu.

### PR DESCRIPTION
**Pull request by idmontie**
Close #136

### CODE REVIEW

File: test/qunit/visual/navigation-test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    equal( true, true, 'an assertion needs to run no matter what' )
+  })
+
+  test('navigation should NOT be visible if mobile', function () {
+    if (window.matchMedia('(max-width: 992px)').matches) {
+      var is_visible = $('.navbar.navbar-ws .navbar-collapse').is(':visible')
+      equal( is_visible, false, 'navigation is NOT visible' )
+    }
+
+    equal( true, true, 'an assertion needs to run no matter what' )
```
Comment (rraub) [inline test/qunit/visual/navigation-test.js]: Why is that?

Comment (idmontie) [inline test/qunit/visual/navigation-test.js]: What? Why is what?

Comment (rraub) [inline test/qunit/visual/navigation-test.js]: Why do we need to make an assertion no matter what?

Comment (idmontie) [inline test/qunit/visual/navigation-test.js]: Karma will fail the test if no assertion is run.

Comment (rraub) [inline test/qunit/visual/navigation-test.js]: Gotcha :ok_hand:

**Pull request merged**